src/main/kotlin/no/nav/familie/ef/sak/arbeidsforhold/ArbeidsforholdService.kt

```diff
@@ -13,9 +12,18 @@ class ArbeidsforholdService(
     private val arbeidsforholdClient: ArbeidsforholdClient,
 ) {
 
-    fun hentArbeidsforhold(fagsakId: UUID, ansettelsesperiodeFom: LocalDate): List<ArbeidsforholdDto> {
+    fun hentArbeidsforhold(fagsakId: UUID, ansettelsesperiodeFom: LocalDate): List<Arbeidsforhold> {
         val aktivIdent = fagsakService.hentAktivIdent(fagsakId)
         val arbeidsforholdResponse = arbeidsforholdClient.hentArbeidsforhold(aktivIdent, ansettelsesperiodeFom)
-        return arbeidsforholdResponse.data?.tilDto() ?: emptyList()
+        return arbeidsforholdResponse.data ?: emptyList()
+    }
+
+    fun finnesAvsluttetArbeidsforholdSisteGittAntallMåneder(aktivIdent: String, antallMåneder: Long = 6): Boolean {
+        val arbeidsforhold = arbeidsforholdClient.hentArbeidsforhold(aktivIdent, LocalDate.now().minusMonths(6)).data
+
+        return arbeidsforhold?.any {
```

Comment:
Hvis ansatt for 9mnd siden og sagt opp for 1mnd siden så får vi "true" her - er det meningen?